packages/starlight/translations/ar.json

```
@@ -13,17 +13,17 @@
   "sidebarNav.accessibleLabel": "الرئيسية",
   "tableOfContents.onThisPage": "على هذه الصفحة",
   "tableOfContents.overview": "نظرة عامة",
-  "i18n.untranslatedContent": "هذا المحتوى لا يتوفر بلغتك بعد.",
+  "i18n.untranslatedContent": "هذا المحتوى ليس متوفرا بلغتك بعد.",
```

Should this be متوفرا or متوفر?
```suggestion
  "i18n.untranslatedContent": "هذا المحتوى ليس متوفر بلغتك بعد.",
```

---

هذا المحتوى غير متوفر بلغتك.

copy pasted from a microsoft page


should we say "this content is not  available yet in your language " or should we remove "yet"?

i removed yet now

---

The English uses “yet”, so if it feels idiomatic, we can keep it in Arabic too.

---

هذا المحتوى غير متوفر بلغتك بعد.
this content is not available in your language yet

this one is perfect the other one you added has grammar error

---

Oops, sorry! Would you like to make a follow-up PR to fix it? I was too enthusiastic merging this one 😁

---

no problem ✨👌👌